src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/MetadataAndSerializationContextTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+
+//using System.Text.Json.Serialization;
+//using Xunit;
+
+//namespace System.Text.Json.SourceGeneration.Tests
```

Comment:
Should we be uncommenting these?

Reply:
https://github.com/dotnet/runtime/pull/53212#discussion_r639993660